packages/terra-action-header/CHANGELOG.md

```diff
@@ -11,14 +11,19 @@ Unreleased
 
 2.15.0 - (June 11, 2019)
 ------------------
+### Added
+* Added CSS background-image variables in .header-icon for back, close, maximize, minimize, previous, and next
```

Comment:
Can we add each variable added and remove so people working on themes knows what changed without looking at the merged PR?

Reply:
Like this? [update](https://github.com/cerner/terra-core/pull/2446/files#diff-b2e7d07bc822cd35b81645ab09c4bdd6)